Sentinel/Classes/DatabaseImportExport/DatabaseImportExportTool.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  DatabaseImportExportTool.swift
+//  Sentinel
+//
+//  Created by Zvonimir Medak on 12.03.2025..
+//
+
+import SwiftUI
+import UniformTypeIdentifiers
+
+/// Tool which gives the ability to import or export the database file
+public struct DatabaseImportExportTool: Tool {
```

Comment:
I would make this `DatabaseTool`, and functions inside are import/export, but in future we can extend that.